src/main/java/org/spongepowered/api/util/ban/Ban.java

```diff
@@ -83,21 +84,42 @@ static Ban of(org.spongepowered.api.entity.living.player.User user, Text.Literal
      *
      * @return The reason specified for the ban.
      */
-    Text.Literal getReason();
+    Text getReason();
 
     /**
-     * Gets the start date of the ban.
+     * Gets the creation date of the ban.
+     *
+     * <p>Note that this {@link Date} has no effect on whether or not a ban is
+     * active. Any ban for which {@link BanService#hasBan(Ban)} returns
+     * <code>true</code> will be used (when checking if a player can join,
+     * for example), regardless of its creation date.</p>
      *
      * @return Creation date of the ban
      */
-    Date getStartDate();
+    Date getCreationDate();
```

Comment:
@ryantheleach: That's true. However, I don't think it makes sense to add this extra complexity to `BanService`, especially when it's never really been considered part of banning in Vanilla. If plugin want to do anything significantly more complicated that what `BanService` provides, they should just cancel `ClientConnectionEvent.Login` as needed.